ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```
@@ -2120,6 +2120,7 @@ ProductInfoSection_new=New...
 ProductInfoSection_app=Application:
 ProductInfoSection_appTooltip=The application extension identifier
 ProductInfoSection_launchers=The product includes native launcher artifacts
+ProductInfoSection_jre=The product includes a JRE
```

A more active label would IMO be more suitable. I was always wondering why the launcher label is so passive.
```suggestion
ProductInfoSection_launchers=Include native launcher artifacts
ProductInfoSection_jre=Include a JRE
```

---

I think it is maybe to make it look a bit nicer in the UI and sounds a bit more friendly, also both flags are only relevant in case one exports the product or for tools like Tycho / pdebuild so they are more like a tag.